internal/services/compute/managed_disk_resource.go

```diff
@@ -839,6 +839,17 @@ func resourceManagedDiskUpdate(d *pluginsdk.ResourceData, meta interface{}) erro
 					shouldShutDown = false
 				case "stopped":
 					shouldShutDown = false
+				case "starting":
+					// send a duplicate start command to ensure Virtual Machine is not in transitioning state which doesn't accept further command
+					log.Printf("[DEBUG] Starting %q", virtualMachine)
+					future, err := vmClient.Start(ctx, virtualMachine.ResourceGroup, virtualMachine.Name)
```

Comment:
Same as above comment

Reply:
`instanceView.Statuses` has some other status information other than `powerState`, and it shall only contains one single entry of the `powerState\starting` or other power state.